llama.h

```diff
@@ -290,6 +290,7 @@ extern "C" {
         bool pure;                           // quantize all tensors to the default type
         void * imatrix;                      // pointer to importance matrix data
         void * kv_overrides;                 // pointer to vector containing overrides
+        bool keep_split;                     // quantize to the same number of shards
```

Comment:
Move the new `bool` after `pure` to keep the bools together. Initialize it to `false` in `llama.cpp: llama_model_quantize_default_params()`

Reply:
Done